packages/css/src/components/accordion/accordion.scss

```diff
@@ -5,25 +5,31 @@
 
 @import "../../common/text-rendering";
 
+.ams-accordion {
+  display: flex;
+  flex-direction: column;
+  gap: var(--ams-accordion-gap);
+
+  @include text-rendering;
+}
+
 .ams-accordion__header {
   display: flex;
   margin-block: 0;
   margin-inline: 0;
 }
 
-@mixin reset-button {
+.ams-accordion__button {
   background-color: transparent;
   border: 0;
-}
-
-.ams-accordion__button {
   color: var(--ams-accordion-button-color);
   cursor: pointer;
   display: flex;
   font-family: var(--ams-accordion-button-font-family);
   font-size: var(--ams-accordion-button-font-size);
   font-weight: var(--ams-accordion-button-font-weight);
-  justify-content: space-between;
+  gap: var(--ams-accordion-button-gap);
+  justify-content: flex-start;
```

Comment:
We can remove this: the initial value is `normal` and “[a]s `stretch` behaves as `start` in flex containers, `normal` also behaves as `start`” ([MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/justify-content#normal)).

```suggestion
```